src/DW-K/BOJ - 10434번 - 행복한 소수/week7.cpp

```diff
@@ -0,0 +1,81 @@
+#include<iostream>
+#include<algorithm>
+#include<cmath>
+#include<vector>
+
+using namespace std;
+
+bool isPrime(int);
+bool happyPrime(int);
+void splitInt(int);
+
+vector<int> v;
+
+int main(void) {
+	ios_base::sync_with_stdio(false);
+	cin.tie(NULL);
+	int N;
+
+	cin >> N;
+
+	int testCaseNum;
+	int num;
+
+	for (int i = 0; i < N; ++i) {
+		cin >> testCaseNum >> num;
+		if (isPrime(num) == false) {
+			cout << testCaseNum << " " << num << " " << "NO\n";
+		}
+		else {
+			if (happyPrime(num) == true) {
+				cout << testCaseNum << " " << num << " " << "YES\n";
+			}
+			else {
+				cout << testCaseNum << " " << num << " " << "NO\n";
+			}
+		}
```

Comment:
```suggestion
		if (happyPrime(num) && isPrime(num)) {
			cout << testCaseNum << " " << num << " " << "YES\n";
		}
		else {
		       cout << testCaseNum << " " << num << " " << "NO\n";
		}
```
cout문의 반복을 줄일 수 있을 것 같습니다.

Reply:
앗 그러네요 감사합니다!